docs/cunumeric/source/comparison/comparison_table.rst.inc

```diff
@@ -0,0 +1,618 @@
+NumPy vs cuNumeric APIs
```

Comment:
Do we need to add this file to the repository? I thought this was auto-generated.

Reply:
Yes, It is auto generated. I will remove it from the repo.